<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>615</width>
    <height>455</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <property name="styleSheet">
   <string notr="true">background-color: rgb(114, 159, 207);</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <widget class="QLabel" name="label">
    <property name="geometry">
     <rect>
      <x>200</x>
      <y>10</y>
      <width>231</width>
      <height>31</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>Arial</family>
      <pointsize>20</pointsize>
      <weight>75</weight>
      <bold>true</bold>
     </font>
    </property>
    <property name="text">
     <string>Lista de Serviços</string>
    </property>
   </widget>
   <widget class="QTableWidget" name="tbl_servicos">
    <property name="geometry">
     <rect>
      <x>30</x>
      <y>80</y>
      <width>551</width>
      <height>281</height>
     </rect>
    </property>
    <property name="minimumSize">
     <size>
      <width>551</width>
      <height>0</height>
     </size>
    </property>
    <property name="sortingEnabled">
     <bool>false</bool>
    </property>
    <column>
     <property name="text">
      <string>Serviço</string>
     </property>
     <property name="font">
      <font>
       <family>Arial</family>
       <pointsize>15</pointsize>
      </font>
     </property>
    </column>
    <column>
     <property name="text">
      <string notr="true">Usuário</string>
     </property>
     <property name="font">
      <font>
       <family>Arial</family>
       <pointsize>15</pointsize>
       <weight>50</weight>
       <bold>false</bold>
      </font>
     </property>
    </column>
    <column>
     <property name="text">
      <string>Senha</string>
     </property>
     <property name="font">
      <font>
       <family>Arial</family>
       <pointsize>15</pointsize>
      </font>
     </property>
    </column>
   </widget>
   <widget class="QPushButton" name="btn_pdf">
    <property name="geometry">
     <rect>
      <x>60</x>
      <y>380</y>
      <width>131</width>
      <height>31</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">color: rgb(0, 0, 0);
background-color: rgb(239, 41, 41);</string>
    </property>
    <property name="text">
     <string>PDF</string>
    </property>
   </widget>
   <widget class="QPushButton" name="btn_excluir">
    <property name="geometry">
     <rect>
      <x>420</x>
      <y>380</y>
      <width>131</width>
      <height>31</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">color: rgb(255, 255, 255);
background-color: rgb(32, 74, 135);</string>
    </property>
    <property name="text">
     <string>Excluir</string>
    </property>
   </widget>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
